instrumentation.go

```diff
@@ -42,6 +48,9 @@ const (
 	// envResourceAttrKey is the key for the environment variable value containing
 	// OpenTelemetry Resource attributes.
 	envResourceAttrKey = "OTEL_RESOURCE_ATTRIBUTES"
+	// envTracesExportersKey is the key for the environment variable value
+	// containing what OpenTelemetry trace exporter to use.
+	envTracesExportersKey = "OTEL_TRACES_EXPORTER"
```

Comment:
I think we should document that additional exporters can be registered via `autoexport` module.

Reply:
SGTM, I'll plan on doing this in a follow-up PR.

Reply:
https://github.com/open-telemetry/opentelemetry-go-instrumentation/issues/437